tests/suites/deploy/deploy_bundles.sh

```diff
@@ -99,14 +99,25 @@ run_deploy_exported_charmhub_bundle_with_float_revisions() {
 	file="${TEST_DIR}/test-export-bundles-deploy-with-float-revisions.log"
 
 	ensure "test-export-bundles-deploy-with-float-revisions" "${file}"
-	bundle=./tests/suites/deploy/bundles/telegraf_bundle_without_revisions.yaml
-	bundle_with_fake_revisions=./tests/suites/deploy/bundles/telegraf_bundle_with_fake_revisions.yaml
+	if [[ -n ${MODEL_ARCH:-} ]]; then
+		bundle=./tests/suites/deploy/bundles/telegraf_bundle_without_revisions_${MODEL_ARCH}.yaml
```

Comment:
@anvial this will only work with `arm64` MODEL_ARCH, right? maybe it's better to check `${MODEL_ARCH} == "arm64"`?

Reply:
Good catch, I need to think if how to make this universal